smol_podcaster.py

```diff
@@ -188,6 +188,37 @@ def tweet_suggestions(transcript):
     suggestions = f"GPT-4 tweet suggestions:\n{gpt_suggestions}\n\nClaude's tweet suggestions:\n{claude_suggestions}\n"
     
     return suggestions
+
+def upload_file_and_use_url(file_path):
+    """
+    Uploads a file to the temporary file hosting service and prints the downloadable file URL.
+
+    Parameters:
+    - file_path: The local path to the file you want to upload.
```

Comment:
Can you add an example? Can people put relative paths or does it need absolute ones? Python paths always mess me up lol

Reply:
Adding it, can do both at least on OSX.